src/test/regress/specs/isolation_add_node_vs_reference_table_operations.spec

```diff
@@ -110,3 +132,4 @@ permutation "s1-begin" "s1-add-second-worker" "s2-insert-to-reference-table" "s1
 permutation "s2-begin" "s2-insert-to-reference-table" "s1-add-second-worker" "s2-commit" "s2-print-content"
 permutation "s1-begin" "s1-add-second-worker" "s2-ddl-on-reference-table" "s1-commit" "s2-print-index-count"
 permutation "s2-begin" "s2-ddl-on-reference-table" "s1-add-second-worker" "s2-commit" "s2-print-index-count"
+permutation "s2-begin" "s2-create-reference-table-2" "s1-add-second-worker" "s2-commit" "s2-print-content-2" "s2-drop-reference-table-2"
```

Comment:
Can you add another permutation where node addition comes first? I believe that's the main issue.